pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -255,6 +257,13 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
     const forced_options = [`subvol=${subvol.pathname}`];
     const mount_point_in_parent = get_mount_point_in_parent(volume, subvol);
 
+    if (client.in_anaconda_mode()) {
+        actions.push({
+            title: _("Edit mount point"),
+            action: () => edit_mount_point(block, forced_options, subvol),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19973-20240208-140838-896b0145-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)